gateway-ha/src/test/java/io/trino/gateway/ha/TestGatewayHaMultipleBackend.java

```diff
@@ -158,6 +158,49 @@ public void testQueryDeliveryToMultipleRoutingGroups()
         assertThat(response4.body().string()).contains("http://localhost:" + routerPort);
     }
 
+    @Test
+    public void testTrinoClusterHostCookie()
+            throws Exception
+    {
+        RequestBody requestBody =
+                RequestBody.create(MediaType.parse("application/json; charset=utf-8"), "SELECT 1");
```

Comment:
Please avoid using deprecated `RequestBody.create` method. Also, pleas fix other waring in this test.

Reply:
All the existing tests use the same. Can you please provide the right method to use here?